tests/utils.py

```diff
@@ -11,6 +11,17 @@
 
 import avro.schema
 
+import ga4gh.protocol as protocol
+
+
+def applyVersion(routes):
```

Comment:
Why the multiple input types here? This is generally considered an anti-pattern in Python. It would be simpler to just have a single string parameter, and handle the 'listification' in client code.